### ISSUE TITLE
Default export not found when exporting as object literal

### ISSUE THREAD

**Issue by santiagoaguiar** [benmosher/eslint-plugin-import#54]
I'm using the following syntax:
```javascript
export { 
    foo as default,
    bar
}
```

And later importing as:
```javascript
import foo, { bar } from "foo";
```

But I'm getting a 'No default export found in module'. Code runs and compiles fine with babel. 

Switching the `foo` module to:
```javascript
export const bar = ..;
export default foo;
```

Works OK. 

Using 0.8.0, with `babel-eslint` 1.4.3 as  parser.

**Comment by benmosher** [comment]
Fascinating. I have never run into this syntax. Esprima is [cool with it too](http://esprima.org/demo/parse.html?code=%2F%2F%20Life%2C%20Universe%2C%20and%20Everything%0Avar%20answer%20%3D%206%20*%207%3B%0A%0Aexport%20%7B%20answer%20as%20default%20%7D%3B).

I would have thought this was invalid, but given [the spec on exports](http://www.ecma-international.org/ecma-262/6.0/#sec-exports-static-semantics-early-errors), seems like this is a normal case. Not without [precedent](http://www.2ality.com/2014/09/es6-modules-final.html#the_default_export_is_just_another_named_export), either.

I'm not totally clear on whether this is valid per the spec (Babel allows a [lot of things](https://github.com/babel/babel/issues/2541)), but it certainly seems useful. 

I'm not sure what to do... seems like the pragmatic thing is to handle it as you've described.

**Comment by benmosher** [comment]
Probing the net with this [StackOverflow post](http://stackoverflow.com/questions/33155785/is-export-foo-as-default-valid-es2015), too.

**Comment by santiagoaguiar** [comment]
I _think_ it should be accepted according to http://www.ecma-international.org/ecma-262/6.0/#table-42.

**Comment by benmosher** [comment]
Yeah, between that and [table 40](http://www.ecma-international.org/ecma-262/6.0/#table-40), I think you're right. I will handle. 😁👍

**Comment by benmosher** [comment]
Let me vet the changes a little tomorrow at work, and then I will publish as 0.8.1. Thanks for the report!

**Comment by santiagoaguiar** [comment]
Amazing!! Thanks a lot for that prompt reply :). 

I used the new version and it worked great for me.
